### ISSUE TITLE
Text in arkanoid example is invisible

### ISSUE THREAD

**Issue by jgarvin** [not-fl3/macroquad#18]
At the start:

```
    // build camera with following coordinate system:
    // (0., 0)     .... (SCR_W, 0.)
    // (0., SCR_H) .... (SCR_W, SCR_H)
    set_camera(Camera2D {
        zoom: vec2(1. / SCR_W * 2., -1. / SCR_H * 2.),
        target: vec2(SCR_W / 2., SCR_H / 2.),
        ..Default::default()
    });
```

But then when text is drawn:

```
            draw_text(
                "Press space to start",
                screen_width() / 2. - 140.,
                screen_height() / 2.,
                30.,
                mq::BLACK,
            );
```

2 issues here:
* draw_text appears to be respecting the camera coordinate system, so screen_width and screen_height are way out of bounds because they are almost always greater than 20
* it also appears to be applying the coordinate transformation to the text **size**, not sure if this is intentional, so 30 is HUGE. I can see the text if I use:

```
        draw_text(
            "Press space to start",
            SCR_W / 2.,
            SCR_H / 2.,
            1.,
            mq::GREEN,
        );
```

I'm compiling to web assembly and viewing in Chrome in case that's relevant.

It would be nice if there was a way for me to draw text knowing that the text wouldn't be stretched or distorted at all, but I don't know how to choose the font size based on the camera parameters so that the original font pixels and the screen exactly line up?

**Comment by not-fl3** [comment]
![imgur-2020_06_21-11:03:26](https://user-images.githubusercontent.com/910977/85229471-757d0e00-b3af-11ea-9ad0-0167ad7f6757.png)

fixed, published `0.2.4`

**Comment by jgarvin** [comment]
Sweet, thanks
